README.md

```diff
@@ -14,6 +14,31 @@ Run `npm test`
 to the following object (`{memory: true}`) being passed in as a parameter for new 
 sqliteDatabase() located in `searchDao.js`, AND all `Spec js` files in the test directory.
 
+## Documentation
+
+When making a GET call, you can add query parameters. These are all of the parameters you 
+can add.
+```
+?class=1&seat=A,B&departureDateAfter=2038-01-11&
+departureDateBefore=2038-01-12&
+arrivalDateAfter=2038-01-16&
+arrivalDateBefore=2038-01-21&
+departure_location=TWS,DSA&
+destination_location=FSE,CAR
+```
+All of the query parameters are optional
+-   `class`: range between 1st class and eco (represented by numbers, 1 meaning first class)
+-   `seat`: give a list of seats that you desire (seperated by commas with no spaces)
```

Comment:
"With no spaces" is generally implicit when talking about query strings (and URLs in general). If spaces *were* allowed, you'd have to talk about how they were encoded, since they're not a character that is allowed to be included in a URL by the relevant Internet standards.

Reply:
Oh interesting. I will add that comment to the documentation